src/calculateRentalCost.js

```diff
@@ -27,8 +27,24 @@
  * @return {number}
  */
 
-function calculateRentalCost(days) {
-  // write code here
-}
+function calculateRentalCost(days = 0) {
+  const sevenDays = 7;
+  const threeDays = 3;
+  const pricePerDay = 40;
+  const discFromSevenDays = 50;
+  const discFromThreeDays = 20;
```

Comment:
У тебя щас четыре из пяти переменных подвязаны к значению, которое в них лежит
```suggestion
  const smallPeriod = 7;
  const threeDays = 3;
  const pricePerDay = 40;
  const discFromSevenDays = 50;
  const smallDiscount = 20;
```
Таким образом тебе не надо будет переименовывать переменные, если поменяется значение внутри(глупо было бы если переменная называется sevenDays, а внутри например 10, а не 7)